demo/ksnav_app_linux/ksnav_kdb.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need to check these JSON in, or is a combination of updating the current documentation with a code section example and tying that into this readme enough?

Reply:
Fixing his readme and json files.  We'd refer to his files in the demo readme above.